src/components/YahooSellNotification.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import SetUserSearchEngineMutation from 'src/utils/mutations/SetUserSearchEngineMutation'
+import SetYahooSearchOptInMutation from 'src/utils/mutations/SetYahooSearchOptInMutation'
+import CreateSearchEnginePromptLogMutation from 'src/utils/mutations/CreateSearchEnginePromptLogMutation'
+import PropTypes from 'prop-types'
+import { Typography } from '@material-ui/core'
+import Notification from './Notification'
+
+const useStyles = makeStyles((theme) => ({
+  learnButton: {
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+  },
+  noThanksButton: {
+    marginLeft: 'auto',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
```

Comment:
This and the "Learn more" button should be font-weight 500:
https://www.figma.com/file/3eY3CAnOdBiQ7eFOP8rnp9/Tab-Legacy%3A-Yahoo-Partnership?node-id=146%3A13055